app/models.py

```diff
@@ -437,6 +473,21 @@ class UserEvent(Event):
     def from_user(cls, user, **kwargs):
         return cls(user_id=user.id, deployment=user.deployment, **kwargs)
 
+class SimpleUserEvent(UserEvent):
+    '''
+    Any kind of user event that doesn't have any extra data associated
+    with it.
+    '''
+
+    # A user completed the registration process and joined the network.
+    SUBTYPE_USER_JOINED = 1
+
+    subtype = Column(types.Integer)
+
+    __mapper_args__ = {
+        'polymorphic_identity': 'simple_user_event'
+    }
```

Comment:
Er, good idea, that actually even further simplifies things by not requiring a db migration at all!